### PR TITLE
NUI-709 NUI developers should not use rendition.target

### PR DESCRIPTION
**Pull request by DrGirlfriend**
## Description

Updated readme to describe renditions a little better and to mention developers not to use `rendition.target`
[NUI-709 NUI developers should not use rendition.target](developers should not use rendition.target)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ Object containing the following attributes:
 | `name` | `string` | filename of the rendition to create |
 | `path` | `string` | Absolute path to store rendition locally (must put rendition here in order to be uploaded to cloud storage) |
 | `index` | `number` | number used to identify a rendition |
-| `target` | `string` or `object` | URL to which the generated rendition should be uploaded or multipart pre-signed URL upload information for the generated rendition |
+| `target` | `string` or `object` | A single URL string to which the generated rendition should be uploaded or an object array containing many multipart pre-signed URLs for the generated rendition </br> While developing, `rendition.target` should not be used |
```
Comment (jdelbick) [inline README.md:130]: Maybe say, 
```
While developing custom workers, `rendition.target` should not be used
```

**Comment by DrGirlfriend** [comment]
Removing `rendition.target` all together per https://cq-dev.slack.com/archives/GAMJLHSBH/p1596497115071300?thread_ts=1596491026.062200&cid=GAMJLHSBH

**Comment by bobvanmanen** [comment]
Maybe there should be a section of "reserved" names?  Basically properties that are used internally by the SDK and are not intended to be used by workers using the SDK?

**Comment by tmathern** [comment]
@bobvanmanen At some point, Alex suggested to omit those "internal/private" names in the docs, and only talk about what is made available.

**Comment by DrGirlfriend** [comment]
What is the verdict on this?

**Comment by bobvanmanen** [comment]
@tmathern @DrGirlfriend Ok, my only thought is that if there is no mention of "reserved" properties customers may decide to introduce their own target property. I don't feel strongly about this though.

**Comment by DrGirlfriend** [comment]
Will need a new ticket in the new sprint for changes.

**Pull request merged**